### ISSUE TITLE
Any paper or document about how to reproduce the results of the pretrained weights?

### ISSUE THREAD

**Issue by stereomatchingkiss** [Deci-AI/super-gradients#1590]
### 💡 Your Question

I would like to retrain the yolo nas S model on the coco dataset, do this project has any document or paper to show us how to replicate the results? Thanks

### Versions

3.3.1

**Comment by BloodAxe** [comment]
We provide a `coco2017_yolo_nas_s.yaml` recipe to train a Yolo-NAS pose from scratch on COCO dataset.
This is NOT recipe to reproduce our results, but a good starting point if you want to train your own version of YoloNAS that is not bound to original YoloNAS weights license.

We trained YoloNAS in a few staged, including pre-training on O365 and further KD on COCO, which is too complicated and at the moment there are no plans to release it.
